tests/test_port.py

```diff
@@ -27,8 +27,8 @@ def test_PortMtu(self, dvs, testlog):
 
     def test_PortNotification(self, dvs, testlog):
 
-        dvs.runcmd("ifconfig Ethernet0 10.0.0.0/31 up") == 0
-        dvs.runcmd("ifconfig Ethernet4 10.0.0.2/31 up") == 0
+        dvs.runcmd("config interface startup Ethernet0 10.0.0.0/31") == 0
+        dvs.runcmd("config interface startup Ethernet4 10.0.0.2/31") == 0
```

Comment:
please separate into two commands. config interface startup only takes the interface name. 

use ```config interface ip add Ethernet0 10.0.0.0/31```

Reply:
Good catch, thank you!